android/engine/src/main/java/org/smartregister/fhircore/engine/domain/model/ProfileData.kt

```diff
@@ -82,11 +82,14 @@ sealed class ProfileData(open val logicalId: String, open val name: String) {
     override val name: String,
     val identifier: String? = null,
     val birthdate: Date,
-    val age: String,
+    val age: String = birthdate.toAgeDisplay(),
     val gender: Enumerations.AdministrativeGender,
     val address: String,
     val phoneContacts: List<ContactPoint> = listOf(),
-    val chwAssigned: Reference
+    val chwAssigned: Reference,
```

Comment:
specifically on chwAssigned, can we manage this through linking a CareTeam to the Patient and putting the Practitioner (CHW) in that CareTeam?

Reply:
need some guidance or sample to map that with careteam

Reply:
@Gental-Giant @pld 

I had a call with @AbdulWahabMemon here are the notes. 

**Notes**
**Questions to answer**
- Are CHWs allowed to assign other CHWs patients. 

**Registration workflow (Assigning to Care teams)**
**FHIR Web**
- Create the CHW (Practitioner)
- Create the patient category groups
- Create the CHW care teams
    - Assign the groups above to the correct care team

**FHIR Core**
**Registration**
- During patient registration, add the patient to the correct patient category group
- On CHW assigned selection, do we show the CHWs on the specific care team or do we show all the CHWs? 

**Profile**
- We can add a section called an assignment. 
    - This should show the care team that is assigned to the group to which the patient is assigned. 



**Registration workflow (Assigning to Practitioner)**
**FHIR Web**
- Create the CHW (Practitioner)

**FHIR Core** 
**Registration**
- During patient registration, assign the patient to the correct practitioner 
- On CHW assigned selection (Not dynamic currently)
    - Show a list of all practitioners in the app. 
    - This list should be synced locally from the server. 

**Profile**
- We can add a section called an assignment. 
   - This should show the practitioner the patient is assigned to. 


@pld Based on the conversation above we might need to add a section on the profile to show the assignments. It should be configurable to show individual assignments or team assignments. 

@Gental-Giant 
- Please check in with the client on the Question above. 
- Also check in with @rowo @HenryRae about the best place to have the assignments shown. 


@pld I think we can leave the implementation as is till we figure out the correct workflow.